non_logged_in_area/templates/base_non_logged_in.html

```diff
@@ -104,6 +104,12 @@
                     <li>
                         <a href="/faq/">{% trans "FAQ" %}</a>
                     </li>
+                    <li>
+                        <a href="https://github.com/coders4help/volunteer_planner" target="_blank">
+                            <img alt="Volunteer-Planner on GitHub" src="{% static 'img/non-logged-in-area/gh-mark.png' %}"
+                            width="auto" height="20"> GitHub
```

Comment:
This the only item in the footer having an icon. Remove icon?

```suggestion
                           GitHub
```

Reply:
Sounds reasonable. Second that. Let's look for nifty icons for all footer links and then bring the GH icon back. :)